### PR TITLE
logistic_consignee: split the module

### PR DESCRIPTION
**Pull request by gurneyalex**
bits are now split in sale_addresses, purchase_addresses,
purchase_requisition_addresses and stock_addresses.
- [x] https://github.com/OCA/purchase-workflow/pull/110
- [x] https://github.com/OCA/stock-logistics-transport/pull/6
- [x] https://github.com/OCA/stock-logistics-transport/pull/4

### CODE REVIEW

**Comment by gurneyalex** [comment]
superseeds #140

**Comment by gurneyalex** [comment]
superseeds #139

**Comment by jgrandguillaume** [comment]
I've tested that and it works fine, So :+1: when build are green

**Comment by lepistone** [comment]
@gurneyalex now that dependencies are merged, could you delete the commit changint `.travis.yml` ?

**Comment by gurneyalex** [comment]
rebased

File: logistic_order_donation_shipment/model/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.depends('picking_id.picking_type_id.code',
+                 'picking_id.picking_type_id.warehouse_id.partner_id',
+                 'picking_id.group_id.procurement_ids.purchase_id.partner_id')
+    @api.one
+    def _get_ship_addresses(self):
+        super(StockMove, self)._get_ship_addresses()
```
Comment (yvaucher) [inline logistic_order_donation_shipment/model/stock_move.py]: I couldn't find where was this super method defined. I searched for it in stock-logistics-transport but it's not there.

Comment (gurneyalex) [inline logistic_order_donation_shipment/model/stock_move.py]: It comes from https://github.com/OCA/stock-logistics-workflow/blob/8.0/stock_shipment_management/model/stock_move.py#L88 

(and without this call the tests are red).

Comment (yvaucher) [inline logistic_order_donation_shipment/model/stock_move.py]: Ok then we will probably need another PR as this method will disappear in https://github.com/OCA/stock-logistics-transport/pull/4 (move from `stock-logistics-workflow` to `stock-logistics-transport`)

Comment (gurneyalex) [inline logistic_order_donation_shipment/model/stock_move.py]: added the missing dependency. I'll update the PR to work with that changed merged too.

**Comment by yvaucher** [comment]
Failing on test:

```
2015-04-09 13:35:44,356 23762 ERROR openerp_test openerp.addons.logistic_order_donation_shipment_test.tests.test_ship_addresses: FAIL
```

**Comment by gurneyalex** [comment]
yvaucher: relaunched the build after olivier dony merged https://github.com/odoo/odoo/pull/4993 -> green :fireworks:

**Pull request merged**